packages/big-design/src/components/Table/HeaderCell/HeaderCell.tsx

```diff
@@ -53,6 +55,25 @@ const InternalHeaderCell = <T extends TableItem>({
     );
   };
 
+  const renderTooltip = () => {
+    if (tooltip && typeof tooltip === 'string') {
+      return (
+        <Tooltip
+          trigger={
+            <Box as="span" marginLeft="xxSmall">
+              <BaselineHelpIcon size="medium" title="Tooltip icon" />
```

Comment:
🤔 So I've been reading [this article (specifically this section)](https://inclusive-components.design/tooltips-toggletips/#inclusivetooltips) about accessibility within tooltips. From what I read, there are a couple of changes we could make:

1. Add an `id` to the `Tooltip` element, and an `aria-labelledby` to the icon element. If we do this approach, we'd need to use a UUID, luckily we have helpers for that.
2. **(Recommended)** Same sort of approach as `1.` but add `aria-describedby` to the icon element instead. You'd use this approach in conjunction with the `title` attribute, but we'd need to make the `title` attribute more descriptive than pointing out it's a "Tooltip icon".

I recommend **2.** but we either need to make the `title` configurable by the user, or have a generic phrase for what the text should read for screen readers.

Reply:
@chanceaclark  Thanks, it looks good, but our Tooltip component does not accept id as a prop to provide it for an element in DOM. If I understood correctly `id` needs to be added to the block of `Tooltip` component. Please correct me if something is wrong.)

Reply:
Here is an example of approach I understood.
`<Tooltip
          id={tooltipId}
          trigger={
            <Box as="span" marginLeft="xxSmall">
              <BaselineHelpIcon
                aria-describedby={tooltipId}
                size="medium"
                title={tooltip}
              />
            </Box>
          }
          placement="right"
        >
          {tooltip}
</Tooltip>`

Reply:
Opened a PR for the `id` stuff here:
https://github.com/bigcommerce/big-design/pull/554

We'll get that merged in today so you can use it.

Reply:
Thats cool, thank you, I am waiting for the merging.

Reply:
@chanceaclark done